### ISSUE TITLE
Panning the screen using two fingers sometimes manipulates sliders and buttons if pointer happens to cross over one

### ISSUE THREAD

**Issue by loganbraywork** [phetsims/molecules-and-light#214]
**Test device**
Hanson
**Operating System**
Windows 7 1809
**Browser**
Edge 44.17763.1.0
**Problem description**
From https://github.com/phetsims/QA/issues/419

Using two fingers to pan up or down or left or right when the mouse is hovered over any slider will manipulate that slider. 
As if you used key navigation to select a slider and then used the left and right arrows to adjust it. 
**Steps to reproduce**
1. zoom in using control +
2. pan around the screen using two fingers on the trackpad
3. position the pointer on a slider or set of buttons and attempt to pan the screen using the trackpad
**Visuals**

![molecules lightedge](https://user-images.githubusercontent.com/54685448/64826270-5ed29c80-d57d-11e9-8597-955d4fc5388c.gif)

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-customZoom.7/phet/molecules-and-light_en_phet.html?a11y
Version: 1.5.0-customZoom.7 2019-08-28 23:16:46 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36 Edge/18.17763
Language: en-US
Window: 1280x607
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Microsoft (Microsoft Edge)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
This seems to be fixed in other work, would be good to verify though.

**Comment by KatieWoe** [comment]
Seems a bit better, but still can happen when scrolling sideways over the light emitter.

**Comment by jessegreenberg** [comment]
Ah, I am seeing this now with the slider! Crazy

**Comment by jessegreenberg** [comment]
It is not, I am not seeing this for other web sliders.

**Comment by jessegreenberg** [comment]
This is fixed for buttons because we changed it so that we can interrupt a button press when we continue to pan the screen. This is not fixed for sliders, because when we begin a pan gesture on a draggable element, we disable this interrupt functionality.

**Comment by jessegreenberg** [comment]
We are coming back to the issue where one cannot detect when a user is using a mouse vs a trackpad: https://stackoverflow.com/questions/10744645/detect-touchpad-vs-mouse-in-javascript

Nor is there a way to do this with proprietary MSGestureEvents (https://msdn.microsoft.com/en-us/windows/desktop/hh441180#properties)

This behavior may be acceptable, would be good to discuss with the team. I don't see any way around it, other than disabling magnification on microsoft browsers.

**Comment by jessegreenberg** [comment]
I just tested on the new Edge beta, and this issue is not happening there. I now feel confident that we shouldn't spend more time on this.

**Comment by jessegreenberg** [comment]
I discussed this issue with @emily-phet and @terracoda this morning. We agreed that it is not worth spending any more time on and it is not worth removing magnification on Edge. It is easy to recover from this issue (move the mouse off of a draggable element), obvious what is happening, and will not be present in the next Edge, we are OK letting this go.
